### ISSUE TITLE
Can't Update to Latest Version

### ISSUE THREAD

**Issue by mm125Y** [datawhores/OF-Scraper#245]
For some reason when I run this command: `pip install ofscraper --upgrade` and launch ofscraper, the version is still set to 3.2.4

I also tried uninstalling it with: `pip uninstall ofscraper` and reinstalling it but it keeps giving me version 3.2.4
Lastly I also tried using this command: `pip install ofscraper==3.3.11` Still nothing.  What am I doing wrong here?

**Comment by cjb900** [comment]
Did you possibly install a pipx version of the scraper. Because what you did should give you the latest version unless there is some other version being detected.

**Comment by mm125Y** [comment]
Yeah I tried doing the same only with pipx and it still didn't work

**Comment by cjb900** [comment]
Well somewhere in your OS is an old install of the scraper getting detected. The only thing i can suggest is to uninstall ofscraper, python, pip, and pipx and delete all the files for the scraper and then reinstall everything and see if that helps.

**Comment by OpenFieldsOfOz** [comment]
When I run install I have to use `pip3 install --force-reinstall ...` to get the install to work if I already have an installation
